### DISCUSSION TITLE
Always choose which project to open on start up

### DISCUSSION THREAD

**Discussion by nerdroid23** [zed-industries/zed#7498] [Feature Requests]
Is there a way to not have the editor always open the last folder I was working in? I'd like to have a "project selector"; where I can choose any of the last folders I've opened instead of the editor defaulting to the last one.

Couldn't find any configuration keys to do that, or at least one to disable the current behavior.

**Comment by omarahmed786** [comment]
Also wondering this!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
